gslib/utils/system_util.py

```diff
@@ -250,11 +250,12 @@ def IsRunningInteractively():
 
 
 def MonkeyPatchHttp():
-  if platform.python_version().startswith('3.6.5'):
-    _MonkeyPatchHttpForPython_3_6_5()
+  py_ver = [int(part) for part in platform.python_version().split('.')]
```

Comment:
Nit: instead of splitting the full version and pulling it apart into a list, you can just do:

```
# Python 3.5.6
sys.version_info.major  # -> 3
sys.version_info.minor  # -> 5
sys.version_info.micro  # -> 6
```

Reply:
Applied this method - good catch @catleeball !